src/components/item/publish/CCLicenseDialog.tsx

```diff
@@ -15,55 +15,58 @@ import CancelButton from '../../common/CancelButton';
 type Props = {
   open: boolean;
   setOpen: (state: boolean) => void;
-  disabled?: boolean;
-  buttonName: string;
   handleSubmit: () => void;
+  isNewDialog?: boolean;
+  handleBack?: () => void;
+  disableSubmission?: boolean;
 };
 
 const CCLicenseDialog = ({
   open,
   setOpen,
-  disabled = false,
-  buttonName,
   handleSubmit,
+  isNewDialog = true,
```

Comment:
I think it's cleaner if you create a new component `CCLicenseModal` that uses `CCLicenseDialog`. I would avoid having `isNewDialog` everywhere.